Dockerfile

```diff
@@ -39,7 +39,7 @@ FROM gcr.io/cloud-marketplace/google/ubuntu1804@sha256:d9fe979ab72ff02f5b5d95c59
 
 # Set the default working directory.
 # This directory binds with the host directory that is to store the output files generated by HarbourBridge.
-WORKDIR /harbourbridge/harbour_bridge_output
+WORKDIR /harbourbridge/spanner_migration_tool_output
 # Create directory to store the HarbourBridge binary. 
 RUN mkdir /harbourbridge/bin
```

Comment:
Do we need to update the names here?

Reply:
I will update the command names, and everything else in docker files in a separate PR.